go/interpreter/lfvm/gas.go

```diff
@@ -58,8 +58,7 @@ func init() {
 }
 
 func initBerlinGasPrice() {
-	// Changed static gas prices with EIP2929.
-	// these prices account for warm access cost
+	// Changed static gas prices with EIP2929
```

Comment:
If you merge #730 before this PR, this comment is inaccurate, since the 100 gas for warm accesses will no longer be charged as part of the static gas costs.

Reply:
I would prefer to merge #730 first, as it introduces and tested a function needed here.